pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -48,7 +50,9 @@ mkDerivation rec {
     qtwebengine
     qtquickcontrols2
     qtgraphicaleffects
+    qtwebsockets
     sqlite
+    libsecret
   ];
 
   qtWrapperArgs = [
```

Comment:
Like you mentioned in the comments, we might not need these `qtWrapperArgs` anymore, if we have `libsecret` in the buildInputs, but I didn't try it myself.

Reply:
It worked, I'll make the other adjustments.